release_notes.md

```diff
@@ -16,6 +16,16 @@ This file tracks release notes for the loop client.
 
 #### New Features
 
+* `--private` flag is now available on the `loop in` command, meaning users can
+  now loop in to private nodes! This was implemented in [#415](https://github.com/lightninglabs/loop/pull/415)
+  and has an implementation of LND's hop hints creation feature for the time
+  being, with the benefit of not creating an extra new invoice for each loop in
```

Comment:
Nice update! I think we can cut the part of "not creating an extra invoice" because that will be confusing to the end user (they don't know our discussions about the dev of this feature). These notes are more targeted at explaining how to use new features in loop than a record for developers (since we can go back and look at the original prs).